javascript/node/README.md

```diff
@@ -0,0 +1,23 @@
+## Substreams Sink Examples - JS (Node)
+
+This example consumes a Substreams package (specifically, the [Ethereum Explorer](https://substreams.dev/streamingfast/ethereum-explorer/v0.1.2)) using the Substreams JS library in NodeJS.
+
+The token is provided through an environment variable:
```

Comment:
```
The [API token](link to doc) is provided through an environment variable:
```

Let’s do the same on other readmes.

Reply:
Maybe we check to have API key instead at some point (not sure the JS lib has been updated).